### ISSUE TITLE
bicop and BiCopSelect return different models  

### ISSUE THREAD

**Issue by waynelapierre** [vinecopulib/rvinecopulib#259]
Thanks for this great package. I am trying to identify a proper copula model for two random variables. The bicop function returns a Joe-Frank copula while the VineCopula::BiCopSelect returns a t-copula. Is it normal?

**Comment by tnagler** [comment]
Yes that can happen, because the copula implementations and optimizer are not identical. It would be easier to say if what you observe is normal, if you would provide a reproducible example.
